### PR TITLE
Rebuild and cleanup

### PR DESCRIPTION
**Pull request by bondarewicz**
- `master` -> `main`
- `0.0.1-beta1` -> `0.2.4`
- abstract urls by using secrets
- allow `ACTIONS_ALLOW_UNSECURE_COMMANDS` to avoid build failures

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -10,30 +10,39 @@ jobs:
           npm install
       - run: |
           npm run all
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: ${{ secrets.ACTIONS_ALLOW_UNSECURE_COMMANDS }}
```
Comment (pearkes) [inline .github/workflows/test.yml:14]: Having some trouble tracking down the documentation for this variable. Can you provide context on what this is doing?

**Review by pearkes — COMMENTED** [review]
Thanks for doing this! Just one small question for you otherwise looks good.

**Comment by jamesproud** [comment]
Is it possible to merge this PR soon?

File: .github/workflows/test.yml (hunk 1 above)
Comment (jamesproud) [reply .github/workflows/test.yml:14]: Currently this error is thrown:
`Unable to process command '::add-path::/opt/hostedtoolcache/waypoint/0.1.0/x64' successfully.`

> The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands

**Comment by pearkes** [comment]
Hi there, thank for submitting this PR – apologies we never merged it. Wanted to let you know we're going to archive this repository in favor of a native integration. See https://github.com/hashicorp/action-waypoint/issues/45#issuecomment-840213375 for more details. Thanks again.

**Pull request closed without merging**